ExampleProject/ExampleProject.csproj

```diff
@@ -13,18 +13,18 @@
   <ItemGroup>
     <PackageReference Include="MessagePack" Version="2.1.165" />
     <PackageReference Include="MessagePack.ImmutableCollection" Version="2.1.165" />
-    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="9.4.1" />
+	<PackageReference Include="Azure.Storage.Blobs" Version="12.12.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
   </ItemGroup>
   <!-- When developing locally, comment out this block and uncomment the one below it. -->
-  <ItemGroup>
+  <!--<ItemGroup>
```

Comment:
The comments should be restored to their original state.

Reply:
Done!, You may want to bump up the `<PackageReference>` versions after publishing this new version to NuGet.